examples/hybrid/sphere/baroclinic_wave_utilities.jl

```diff
@@ -303,9 +319,10 @@ function saturated_surface_conditions(
     params,
     coupled,
 )
-    Cd = surface_scheme.Cd
-    Ch = surface_scheme.Ch
-    # parameters
+    Cd = parent(p.Cd[colidx])[1]
```

Comment:
If I don't do this I got `Cannot convert an object of type Field{} (trunc disp) to an object of type Float32`

Reply:
Hmm would it be better to pass in Cd and Ch as en explicit argument to `saturated_surface_conditions`? Maybe in the long term, we can separate the cache into fields and other objects, so we can just pass in `p.fields_vector[colidx]`, but that change may break many things, so probably not worth it right now.

Reply:
I think we shouldn't ever need parent anymore